vgen/src/main/resources/io/gapi/vgen/go/main.snip

```diff
@@ -210,8 +210,8 @@
 
                     {@methodComment(method, methodName)}
                     func (client *Client) {@methodName}(ctx context.Context, req {@inTypeName}, opts ...gax.CallOption) (*{@iteratorTypeName}, error) {
-                    	callOpts := append({@retryCodesMethodName}RetryCodes(), {@retryParamsMethodName}RetryParams(), client.CallOptions["{@methodName}"]...)
-                    	callOpts = append(callOpts, opts...)
+                    	defaultCallOpts := append({@retryCodesMethodName}RetryCodes(), {@retryParamsMethodName}RetryParams()...)
+                    	callOpts := append(client.CallOptions["{@methodName}"], opts...)
```

Comment:
Not super satisfied with all the appends thrown around... but I couldn't think of a better way (other than making a function that takes an arbitrary number of `[]CallOption`).

LMK